qa/TL1_tensorflow-dali_test/test.sh

```diff
@@ -13,7 +13,7 @@ do_once() {
     CUDA_VERSION=$(echo $(nvcc --version) | sed 's/.*\(release \)\([0-9]\+\)\.\([0-9]\+\).*/\2\3/')
 
     # install TF 2.x
```

Comment:
```suggestion
    # install TF 2.2.*
```

Reply:
Done

Reply:
Isn't it 2.3.1 now? Or am I missing something?

Reply:
You are right. Fixed again.